src/components/utils/aggrid-rows-handler.ts

```diff
@@ -34,7 +34,7 @@ export function getNoRowsMessage(
     }
 }
 
-export function getRows(rows: any[], status: string): any[] {
+export function getRows(rows: any[] | undefined, status: string): any[] {
```

Comment:
I'm not sure we should change function definition in order to make it match our use case

here it should be fine since both methods seem to have been made to work with "undefined"

Reply:
Based on the function definition I also handled the case where rows is undefined so non worries.